pkg/common/common.go

```diff
@@ -7,17 +7,18 @@ import (
 )
 
 const (
-	Velero                       = "velero"
-	Restic                       = "restic"
-	VeleroNamespace              = "oadp-operator"
-	OADPOperator                 = "oadp-operator"
-	OADPOperatorVelero           = "oadp-operator-velero"
-	DataMover                    = "volume-snapshot-mover"
-	DataMoverController          = "data-mover-controller"
-	DataMoverControllerContainer = "data-mover-controller-container"
-	OADPOperatorServiceAccount   = "openshift-adp-controller-manager"
-	VolSyncDeploymentName        = "volsync-controller-manager"
-	VolSyncDeploymentNamespace   = "openshift-operators"
+	Velero                           = "velero"
+	Restic                           = "restic"
+	VeleroNamespace                  = "oadp-operator"
+	OADPOperator                     = "oadp-operator"
+	OADPOperatorVelero               = "oadp-operator-velero"
+	DataMover                        = "volume-snapshot-mover"
+	DataMoverController              = "data-mover-controller"
+	DataMoverControllerContainer     = "data-mover-controller-container"
+	OADPOperatorServiceAccount       = "openshift-adp-controller-manager"
+	VolSyncDeploymentName            = "volsync-controller-manager"
+	VolSyncDeploymentNamespace       = "openshift-operators"
+	DefaultRestoreResourcePriorities = "securitycontextconstraints,customresourcedefinitions,namespaces,storageclasses,volumesnapshotclass.snapshot.storage.k8s.io,volumesnapshotcontents.snapshot.storage.k8s.io,volumesnapshots.snapshot.storage.k8s.io,datauploads.velero.io,persistentvolumes,persistentvolumeclaims,serviceaccounts,secrets,configmaps,limitranges,pods,replicasets.apps,clusterclasses.cluster.x-k8s.io,endpoints,services,-,clusterbootstraps.run.tanzu.vmware.com,clusters.cluster.x-k8s.io,clusterresourcesets.addons.cluster.x-k8s.io"
```

Comment:
after debugging with wes, we think there could be a typo/wrong order in here. What you think?

Reply:
Thanks for the help debugging @kaovilai !
Thanks @shubham-pampattiwar for finding the missing CR!